shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/java/org/apache/shardingsphere/sql/parser/integrate/jaxb/domain/SQLParserTestCases.java

```diff
@@ -258,6 +263,7 @@
         putAll(showCreateTableTestCases, result);
         putAll(showTableStatusTestCases, result);
         putAll(showIndexTestCases, result);
+        putAll(setVariableStatementTestCases, result);
```

Comment:
Please do not interrupt `showXXXTestCases`, please move `setVariableStatementTestCases` behind `showTestCases`